BlackNectar/BlackNectar/StoresDataStruct.swift

```diff
@@ -14,15 +14,16 @@ import UIKit
 // Creates data structure for JSON Request
 struct Store {
     
+    let storeID: String
     let storeName: String
     let location: CLLocationCoordinate2D
     let address: Address
     let storeImage: URL
```

Comment:
self.storeImage was assigned storeImageURL in the initialization
If I assigne self.storeImage to storeImage I have to change storeImage to a var